service/src/test/java/bio/terra/pipelines/controller/JobApiUtilsTest.java

```diff
@@ -1,32 +1,53 @@
 package bio.terra.pipelines.controller;
 
+import static bio.terra.pipelines.app.controller.JobApiUtils.*;
 import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
 
 import bio.terra.common.exception.ErrorReportException;
+import bio.terra.pipelines.app.configuration.external.IngressConfiguration;
 import bio.terra.pipelines.app.controller.JobApiUtils;
 import bio.terra.pipelines.dependencies.stairway.JobMapKeys;
+import bio.terra.pipelines.dependencies.stairway.JobService;
 import bio.terra.pipelines.dependencies.stairway.exception.InternalStairwayException;
 import bio.terra.pipelines.dependencies.stairway.exception.InvalidResultStateException;
 import bio.terra.pipelines.dependencies.stairway.model.EnumeratedJobs;
 import bio.terra.pipelines.generated.model.ApiErrorReport;
 import bio.terra.pipelines.generated.model.ApiGetJobsResponse;
 import bio.terra.pipelines.generated.model.ApiJobReport;
+import bio.terra.pipelines.testutils.BaseEmbeddedDbTest;
 import bio.terra.pipelines.testutils.StairwayTestUtils;
 import bio.terra.pipelines.testutils.TestUtils;
 import bio.terra.stairway.FlightMap;
 import bio.terra.stairway.FlightState;
 import bio.terra.stairway.FlightStatus;
 import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.springframework.http.HttpStatus;
 
-class JobApiUtilsTest {
+class JobApiUtilsTest extends BaseEmbeddedDbTest {
+
+  @InjectMocks private JobApiUtils jobApiUtils;
+  @Mock private JobService jobService;
+  @Mock private IngressConfiguration ingressConfiguration;
+
+  // we mock IngressConfiguration.getDomainName() to return "localhost", which maps to "http://"
```

Comment:
you probably dont have to mock this if you add it to the application-test.yml.  You should be able to `@Autowired` it for any test class taht subclassed BaseEmbeddedDbTest.

At least im pretty sure this is possible, if you've tried it already and it didnt work then ignore :)